test/puppetlabs/puppetdb/http/nodes_test.clj

```diff
@@ -425,6 +425,19 @@
           (are-error-response-headers headers)
           (is (re-find msg body)))))))
 
+;TODO fix all skipped tests
+(deftest-http-app ^:skipped tests-for-known-issues
+  [[version endpoint] endpoints
+  method [:get :post]]
+
+  (testing "in a reports subquery"
```

Comment:
This issue is covered by [PDB-5026](https://tickets.puppetlabs.com/browse/PDB-5026). Adding a comment with the ticket number here should help us remember to enable this test once the issue if fixed.